org-formation/050-costs/_tasks.yaml

```diff
@@ -0,0 +1,13 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+AnomalyDetectorService:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.3/templates/Cost/anomaly-detector-service.yaml
+  StackName: !Sub '${resourcePrefix}-anomaly-detector-service'
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion
+  Parameters:
```

Comment:
Is there a way of setting the `Threshold` parameter on a per-account basis? Like what's currently being done with the budget alerts.

Reply:
Do you mean per AWS account or per IAM account?  My understanding is this will monitor per AWS service in each AWS account.

Reply:
Sorry, I was referring to each account deployed in organizations-infra. Or is this monitor set once on the parent account (_i.e._ the account at the head of the AWS Organization) and cannot be tailored for member accounts (_i.e._ individual accounts in the AWS Organization, like nextflow-prod and nextflow-dev)?

Reply:
That's good to know. I'll have to see how sensitive the Cost Anomaly Detection is before adjusting the threshold for the Nextflow AWS accounts.